README.md

```diff
@@ -10,23 +10,28 @@ Install the library by running:
 composer require openeuropa/epoetry-client
 ```
 
-To create a client instance use the [`\OpenEuropa\EPoetry\EPoetryClientFactory`](./src/EPoetryClientFactory.php):
+To create a client instance use the [`\OpenEuropa\EPoetry\Request\ClientFactory`](src/Request/RequestClientFactory.php):
 
 ```php
 <?php
-use Http\Discovery\HttpClientDiscovery;
-use OpenEuropa\EPoetry\EPoetryClientFactory;
-use OpenEuropa\EPoetry\Type\CreateRequests;
+use GuzzleHttp\Client as GuzzleClient;
+use Http\Adapter\Guzzle6\Client as GuzzleAdapter;
+use OpenEuropa\EPoetry\Request\ClientFactory;
+use OpenEuropa\EPoetry\Request\Type\CreateRequests;
+
+// Instantiate HTTP client.
+$guzzle = new GuzzleClient();
+$adapter = new GuzzleAdapter($guzzle);
 
 // Instantiate the client factory.
-$factory = new EPoetryClientFactory('http://europa.eu/epoetry.wsdl', $adapter);
+$factory = new RequestClientFactory('http://europa.eu/epoetry.wsdl', $adapter);
```

Comment:
This is actually wrong, the `RequestClientFactory` expects the service endpoint and not the WSDL URI.

Reply:
True but out of scope. It also requires to change the command in `\OpenEuropa\EPoetry\Console\Command\CreateRequestsCommand` and it may require changes in other places. It will be better addressed in another PR.